ar_exercise/exercise.rb

```diff
@@ -51,21 +69,51 @@ class Apartment < ActiveRecord::Base
 # Hint, the following methods will help: `new`, `create`, `save`, `uddate`, `destroy`
 
 # Create 3 new apartments, and save them to the DB
+# apt1 = Apartment.new(address: "123 Lollipop Lane", monthly_rent: 3000, sqft: 800, num_beds: 2, num_baths: 2)
+# apt1.save
+# apt2 = Apartment.new(address: "4900 Elm Street", monthly_rent: 2100, sqft: 400, num_beds: 1, num_baths: 1)
+# apt2.save
+# apt3 = Apartment.new(address: "5 Backwoods Place", monthly_rent: 2000, sqft: 5000, num_beds: 22, num_baths: 4)
+# apt3.save
+
 # Create at least 9 new tenants and save them to the DB. (Make sure they belong to an apartment)
+# tenant1 = Tenant.create(name: "Pookins Magoo", age: 45, gender: "Female", apartment_id: 2)
```

Comment:
You shouldn't use hard coded IDs in a script, as they are subject to change (esp if you re-seed your DB, they may not start at 1). Instead do something like this:

```ruby
tenant1 = Tenant.create(name: "Pookins Magoo", age: 45, gender: "Female", apartment: apt1)
# or this works too
tenant1 = Tenant.create(name: "Pookins Magoo", age: 45, gender: "Female", apartment_id: apt1.id)
# or even this!
apt1.tenants.create(name: "Pookins Magoo", age: 45, gender: "Female")
```